net/crowdsec-firewall-bouncer/Makefile

```diff
@@ -0,0 +1,122 @@
+#
+# Copyright (C) 2021 Gerald Kerma
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=crowdsec-firewall-bouncer
+PKG_VERSION:=0.0.13
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/crowdsecurity/cs-firewall-bouncer
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_SOURCE_DATE:=20210705
+PKG_MIRROR_HASH:=7490a99288246bbc883384c47b441bb2f3867724a9f1a0e94d352fe36d47c220
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+CSFB_BUILD_VERSION?=$(PKG_SOURCE_VERSION)
+CSFB_BUILD_GOVERSION=$(shell go version | cut -d " " -f3 | sed -r 's/[go]+//g')
+CWD_BUILD_TIMESTAMP:=$(SOURCE_DATE_EPOCH)
+CSFB_BUILD_TAG=openwrt
+
+CSFB_VERSION_PKG:=github.com/crowdsecurity/cs-firewall-bouncer/pkg/version
+
+GO_PKG:=github.com/crowdsecurity/cs-firewall-bouncer
+
+GO_PKG_LDFLAGS_X:=$(CSFB_VERSION_PKG).Version=$(CSFB_BUILD_VERSION) \
+	$(CSFB_VERSION_PKG).BuildDate=$(CSFB_BUILD_TIMESTAMP) \
+	$(CSFB_VERSION_PKG).Tag=$(CSFB_BUILD_TAG) \
+	$(CSFB_VERSION_PKG).GoVersion=$(CSFB_BUILD_GOVERSION)
+
+GO_PKG_INSTALL_ALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/crowdsec-firewall-bouncer/Default
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Firewall bouncer for Crowdsec.
+  URL:=https://github.com/crowdsecurity/cs-firewall-bouncer/
+endef
+
+define Package/crowdsec-firewall-bouncer
+$(call Package/crowdsec-firewall-bouncer/Default)
+  DEPENDS:=$(GO_ARCH_DEPENDS) \
+  	+crowdsec
+endef
+
+define Package/golang-crowdsec-firewall-bouncer-dev
+$(call Package/crowdsec-firewall-bouncer/Default)
+$(call GoPackage/GoSubMenu)
+  TITLE+= (source files)
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+  PKGARCH:=all
+endef
+
+define Package/crowdsec-firewall-bouncer/Default/description
+  Crowdsec bouncer written in golang for firewalls.
+
+  crowdsec-firewall-bouncer will fetch new and old decisions
+  from a CrowdSec API to add them in a blocklist used by supported firewalls.
+
+  You must install iptables+ipset or nftables.
```

Comment:
Why there are not listed as dependencies?

Reply:
They were, but I removed because of a past review...
We decide to not force the firewall, but to detect the one already present and enable it in CrowdSec.

Reply:
You can not use something like this: `DEPENDS:=@(PACKAGE_iptables||PACKAGE_nftables)`?

Reply:
Sure...
I add this !
Thanks

Reply:
@BKPepe : 
We need to have iptables+ipset or nftables
Can we do something like this with the DEPENDS ?

DEPENDS:=@(PACKAGE_iptables||PACKAGE_nftables)?

Can I then add an IF condition if iptables is select/present to add ipset ?

Reply:
While this is a good addition, I think this only adds a build-time dependency on iptables/nftables. I don't believe this dependency is present at run-time, i.e. when installing with opkg.

Reply:
```
root@ULTRA-5G:~/custom/1.2.0# opkg info crowdsec-firewall-bouncer
Package: crowdsec-firewall-bouncer
Version: 0.0.15-3
Depends: libc, crowdsec
Status: install user installed
Architecture: aarch64_cortex-a53
Conffiles:
 /etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml d818d660a03e3e920451946d53405c067514c94c02b17a1d17a125db3920c627
Installed-Time: 1632124581
```
I confirm this is only a build-time dependency... not a install one !

Reply:
It is still preferable to have the user manually install and choose a such huge package than a FireWall.